src/net/http.rs

```diff
@@ -3756,6 +3817,21 @@ impl HttpResponseType {
             HttpResponseType::ServiceUnavailable(_, ref msg) => {
                 self.error_response(fd, 503, msg)?
             }
+            HttpResponseType::GetHealthError(ref md, ref data) => {
+                HttpResponsePreamble::new_serialized(
+                    fd,
+                    512,
```

Comment:
HTTP 512 (and HTTP 513 below) aren't standard HTTP status codes.  Do you want to put something more standard here, or is there a downstream tool that expects these?

Reply:
Also, `GetHealthError` reports that the node is still bootstrapping, right?  Why is this an internal server error if the node is still bootstrapping?  Wouldn't this still be a 200 OK, since the node's bootstrapping status is an application-level concern above the HTTP transport layer?